05-services.md

```diff
@@ -1243,7 +1247,7 @@ networks:
 
 ### mac_address
 
-`mac_address` sets a MAC address for the service container.
+_DEPRECATED: use [networks.mac_address](#mac_address)_
```

Comment:
We can't mark it as deprecated if it's not also deprecated on the Docker CLI either